Makefile

```diff
@@ -1,7 +1,7 @@
 SRC   = .
 BUILD = build
 
-IVERILOG = iverilog -I $(SRC)
```

Comment:
@piranna are you familiar with Verilator? I was thinking to use it to test the code.